C:\Users\tzwar\AppData\Local\Programs\Python\Python312\python.exe: No module named celery
